sub-packages/bionemo-evo2/src/bionemo/evo2/run/train.py

```diff
@@ -646,7 +645,7 @@ def train(args: argparse.Namespace) -> nl.Trainer:
     ]
 
     if args.lora_finetune:
-        callbacks.append(ModelTransform())
+        callbacks.append(lora_transform)
```

Comment:
in predict.py, lora transformed is initlised by 

```
        if lora_checkpoint_path:
             model_transform = Evo2LoRA(peft_ckpt_path=str(lora_checkpoint_path))
             callbacks.append(model_transform)
        else:
             model_transform = None
```
and there is no need for params lora_finetune. Could we keep it consistent?

Reply:
Imho --lora-finetune is not needed, lets remove it and use lora if the checkpoint is provided. please specify in docs that providing checkpoint enables LoRa